kombuworker/agnostic.py

```diff
@@ -3,11 +3,14 @@
 import json
 import time
 import signal
+
 from types import SimpleNamespace
-from typing import Optional, Callable
+from typing import Optional, Callable, List
```

Comment:
It's standard in 3.9+ to use the constructor names instead of these type names. We should instead add
```
from __future__ import annotations
```
if you're getting errors with older versions of python.

Reply:
Great point